opencti-worker/src/push_handler.py

```diff
@@ -30,10 +35,36 @@ def __post_init__(self) -> None:
             json_logging=self.json_logging,
             ssl_verify=self.ssl_verify,
         )
+        # Small hack to handle any rabbit prefix that could have been configured
+        self.listen_too_large_routing = self.push_routing.replace(
+            "push_routing", "listen_routing"
+        ).replace(self.connector_id, "too-large-bundle")
```

Comment:
look a bit too hacky for me :) couldn't we send the dead letter queue name in the connector config we received from graphql ?

(reposted after rebase)